exporter/debugexporter/README.md

```diff
@@ -45,6 +47,11 @@ exporters:
     verbosity: detailed
     sampling_initial: 5
     sampling_thereafter: 200
+    use_internal_logger: false
+    output_paths:
```

Comment:
I'm not sure if it's needed to be able to set multiple output paths or just one path would suffice, but I suppose we can leave it as is.

I suppose one practical scenario is to send output to both `stdout` and a file on disk.

Reply:
Works for me :+1: